### PR TITLE
Chore: Release notes 2021.01

### LINKED ISSUE #5380: release notes 2021.01
release notes 2021.01

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5380

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,103 @@ id: Release notes
...
+  - Fixed to stop panel content unmounting before collapse ([#5330](https://github.com/patternfly/patternfly-react/pull/5330))
+- **DualListSelector:** 
+  - Added type=button to DualListSelectorListItem button ([#5266](https://github.com/patternfly/patternfly-react/pull/5266))
+  - Fixed to not pass custom handlers to root div ([#5308](https://github.com/patternfly/patternfly-react/pull/5308))
+- **Empty state** 
+  - Cleaned up errors in tests ([#5302](https://github.com/patternfly/patternfly-react/pull/5302))
+- **Jumplinks:** 
+  - Updated to allow hrefs with spaces ([#5223](https://github.com/patternfly/patternfly-react/pull/5223))
+  - **Menu:** 
+    - Fixed sentence case for menu example ([#5262](https://github.com/patternfly/patternfly-react/pull/5262))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:50]: Remove tab on these lines

**Review by redallen — CHANGES_REQUESTED** [review]
I just see the formatting on these 2 lines

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,103 @@ id: Release notes
...
+  - Added new code editor component ([#5294](https://github.com/patternfly/patternfly-react/pull/5294))
+- **Context selector:** 
+  - Removed isHovered from item unit tests ([#5306](https://github.com/patternfly/patternfly-react/pull/5306))
+- **Drawer:** 
+  - Added resize callback ([#5319](https://github.com/patternfly/patternfly-react/pull/5319))
+  - Fixed dragging problem on iframe when resizing drawer ([#5331](https://github.com/patternfly/patternfly-react/pull/5331))
+  - Improved cursor responsiveness ([#5328](https://github.com/patternfly/patternfly-react/pull/5328))
+  - Fixed initial width not respecting min/max widths ([#5313](https://github.com/patternfly/patternfly-react/pull/5313))
+  - Fixed to stop panel content unmounting before collapse ([#5330](https://github.com/patternfly/patternfly-react/pull/5330))
+- **DualListSelector:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:42]:
```suggestion
- **Dual list selector:** 
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,103 @@ id: Release notes
...
+  - Fixed crash when opened using enter with disabled options ([#5258](https://github.com/patternfly/patternfly-react/pull/5258))
+  - Cleaned up errors in tests ([#5302](https://github.com/patternfly/patternfly-react/pull/5302))
+- **Simple list:** 
+  - Added uncontrolled Simple list support ([#5134](https://github.com/patternfly/patternfly-react/pull/5134))
+- **Slider:** 
+  - Added new slider component ([#5358](https://github.com/patternfly/patternfly-react/pull/5358))
+- **Toggle group:** 
+  - Fixed bug with ToggleGroupItem button being clicked in form in modal closing modal ([#5259](https://github.com/patternfly/patternfly-react/pull/5259))
+- **Toolbar:** 
+  - Added log viewer toolbar example ([#5196](https://github.com/patternfly/patternfly-react/pull/5196)
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:71]:
```suggestion
  - Added log viewer toolbar example ([#5196](https://github.com/patternfly/patternfly-react/pull/5196))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,103 @@ id: Release notes
...
+
+### Topology
+  - Adjusted topology node layout when new nodes are added ([#5249](https://github.com/patternfly/patternfly-react/pull/5249))
+ - Prevented context sub menus from overflowing viewport ([#5271](https://github.com/patternfly/patternfly-react/pull/5271))
+
+### Other
+- **Chore:** 
+  - Added linting rule to disallow nested ternaries ([#5255](https://github.com/patternfly/patternfly-react/pull/5255))
+- **Demos:** 
+  - Enable dstatic drawer open on mobile view in primary detail demo ([#5311](https://github.com/patternfly/patternfly-react/pull/5311))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:97]:
```suggestion
  - Enabled static drawer open on mobile view in primary detail demo ([#5311](https://github.com/patternfly/patternfly-react/pull/5311))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
3 small typos in Dual list selector, Toolbar, and Demos

**Review by redallen — DISMISSED** [review]

**Review by redallen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**